src/Sparrow/Json/AbstractBlittableJsonTextWriter.cs

```diff
@@ -214,8 +214,25 @@ public int WriteDateTime(DateTime value, bool isUtc)
             return size;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public void WriteString(string str, bool skipEscaping = false)
         {
+            if (str == null)
+            {
+                WriteNull();
+                return;
+            }
+
+            if (str.Length == 1)
+            {
+                char b = str[0];
+                if (b < 32 || (b >= 127 && b <= 159))  // is a control code character?
```

Comment:
Can we have `32`, `127` and `159` defined as constants with meaningful names?

Reply:
I ended up reverting it, there is no good way (effective) to achieve what I wanted to achieve without tripping on one of the errors showcase at the Slow Tests.